cpp/modules/deck.gl/layers/src/line-layer/line-layer-vertex.glsl.h

```diff
@@ -38,29 +44,12 @@ layout(location = 3) in vec4 instanceColors;
 layout(location = 4) in float instanceWidths;
 
 // TODO(ilija@unfolded.ai): Revisit once double splitting is in place
-vec3 instanceSourcePositions64Low = vec3(0.0);
-vec3 instanceTargetPositions64Low = vec3(0.0);
+vec3 instanceSourcePositions64Low = ZERO_64_LOW;
```

Comment:
What are these constants that are not defined in this file? Does not seem to improve readability to me.